OrbitLinuxTracing/PerfEvent.h

```diff
@@ -369,6 +369,21 @@ class TaskRenamePerfEvent : public TracepointPerfEvent {
   }
 };
 
+class TracepointEventPidTidTimeCpu {
```

Comment:
If this class doesn't need to inherit from `TracepointPerfEvent`, it should at least be part of the `PerfEvent` hierarchy and hence inherit from it, like all the other classes here. Also I think it should be called `GenericTracepointPerfEvent`: all these classes follow the `...PerfEvent` naming pattern.